Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBGridFSBucket-openDownloadStreamByName.txt

MongoDB\\GridFS\\Bucket::openDownloadStreamByName()
===================================================

.. meta::

>    :description: Access a GridFS file by its filename and open it as a readable stream using `openDownloadStreamByName()` in PHP.

Reviewer: ```suggestion
   :description: Access a GridFS file by its filename and open it as a readable stream with the MongoDB PHP Library.
```
